examples/easit-oauth-integration/EasitTestOauthDatastore.js

```diff
@@ -126,6 +134,16 @@ fluid.defaults("gpii.oauth2.easitSampleDataStore", {
                 selectedPreferences: { "": true },
                 revoked: false
             }
+        ],
+        credentialClientTokensIdSeq: 2,
+        credentialClientTokens: [
+            {
+                id: 1,
+                clientId: 6,
+                accessToken: "first_discovery_access_token",
+                revoked: false,
+                allowAddPrefs: true
+            }
```

Comment:
Please remove this credentialClientTokens entry for the First Discovery Tool. In this case, we want the token to be generated, rather than pre-configured.